packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -900,3 +910,34 @@ void WebView::HandleCookieMethodCall(
     result->NotImplemented();
   }
 }
+
+FlutterDesktopGpuBuffer* WebView::CopyGpuBuffer(size_t width, size_t height) {
+  if (!candidate_surface_) {
+    return nullptr;
+  }
+
+  std::lock_guard<std::mutex> lock(mutex_);
+  if (rendered_surface_) {
+    tbm_surface_destroy(rendered_surface_);
+    rendered_surface_ = nullptr;
+  }
+
+  rendered_surface_ = candidate_surface_;
+  candidate_surface_ = nullptr;
+
+  FlutterDesktopGpuBuffer* gpuBuffer = new FlutterDesktopGpuBuffer();
```

Comment:
Snake casing is expected for `gpuBuffer` and `textureVariant_`.

Reply:
:ok_man: